code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -175,9 +146,9 @@
 
 	return cached_results[id] ? REACTING : NO_REACTION
 
-//fusion: a terrible idea that was fun to try. turns co2 and plasma into REALLY HOT oxygen and nitrogen. super exothermic lol
+//fusion: a terrible idea that was fun but broken. Now reworked to be less broken and more interesting.
```

Comment:
a quick rundown on what's actually going on here would be nice, like comments or something